rapthor/pipeline/parsets/image_pipeline.cwl

```diff
@@ -245,9 +245,9 @@ inputs:
     type: float[]
 
   - id: wsclean_mem
-    label: Memory fraction
+    label: Memory percentage
     doc: |
-      The memory fraction limit for WSClean (length = n_sectors).
+      The memory limit for WSClean in percent of total (length = n_sectors).
```

Comment:
Do you still want to keep this a float, now that you changed fraction to percentage?

Reply:
Good point -- an int would work fine too, but I don't think it really matters in practice. By "length = n_sectors", I just mean the length of the input list should be n_sectors long, where n_sectors is the number of sectors; however, it's not really necessary to have a list here for the memory limit (as you probably don't need a different limit for each sector), but it does no harm either I guess.